config/initializers/secure_headers.rb

```diff
@@ -1,5 +1,9 @@
 ::SecureHeaders::Configuration.default do |config|
-  config.hsts = "max-age=#{20.years.to_i}; includeSubdomains"
+  if SETTINGS[:hsts_enabled]
+    config.hsts = "max-age=#{20.years.to_i}; includeSubdomains"
+  else
+    config.hsts = "max-age=0; includeSubdomains"
```

Comment:
Do browsers actually update their HSTS caches when they see this?

Reply:
if they follow the specs, then yes: https://tools.ietf.org/html/rfc6797#section-6.1.1